calico-enterprise/multicluster/federation/kubeconfig.mdx

```diff
@@ -2,15 +2,15 @@
 description: Configure a local cluster to pull endpoint data from a remote cluster.
 ---
 
-# Configure federated endpoint identity
+# Configure federated endpoint identity and multi-cluster networking
 
 ## Big picture
 
-Configure a local cluster to pull endpoint data from a remote cluster.
+Create RemoteClusterConfiguration to federate endpoint identities and establish cross-cluster connectivity.
```

Comment:
The Big picture should not introduce a specific resource or implementation. User is just trying to see if they are on the right task. How about?

Federate endpoint identities and establish cross-cluster connectivity.

Reply:
Took your suggestion, but just added that we are "Configuring a cluster..." for ^